dev-libs/device-atlas-api-c/Manifest

```diff
@@ -1 +1 @@
-DIST deviceatlas-enterprise-c-2.1.zip 504286 SHA256 a02fc29486d245b1c8d946c663b71ba981e88fee8d037f5bc1bf255deb287974 SHA512 99a8e89f3d1c084a93b184685108ea65d1fd925e0c8b52599a42dbe70af3126103da0a9fd284b14ddf59b996204334d360a12651025413bc4d7f76054779275b WHIRLPOOL 939733a58f352cbeee852ccfea3f2f022310ceb7363a15889d9a6366532102ee700271b6008e0a57ae4d86cafe6354409f9520105e46eaabb5c41f1a7827f1b4
+DIST deviceatlas-enterprise-c-2.1.1.tar.gz 197979 SHA256 104d50621533963ce3bd2505daf155e5fb8817671261ff2de0512971cedd7235 SHA512 b2af793ff5bd3b0095d924260cb123bfc0ffaacfd49bb626109af00fa9b6d5532bcb6ea992bd2e64f99b78f26ee0fbe3045784eb16142c56cb1a9bc2a79a6c52 WHIRLPOOL 391c262cb888d7602fafe820c51c1169912108beaec56520a2a534412fd3bd252b5ff7c8aef80ceb227c32444a1ce281487f200421fa4cb309d4c3f538b55022
```

Comment:
Since you are now using a tar.gz, you'll have to remove unpacker eclass from the ebuild